go/vt/vtgate/gorpcvtgateconn/conn_test.go

```diff
@@ -40,16 +43,67 @@ func initEnv() {
 			vttest.LocalTeardown()
 			panic(err)
 		}
+		fmt.Println("Launched local cluster")
 	})
 }
 
-/*
 func TestMain(m *testing.M) {
 	r := m.Run()
 	vttest.LocalTeardown()
 	os.Exit(r)
 }
-*/
+
+func TestExecuteShard(t *testing.T) {
+	t.Skip("Take too much time to run, skipping...")
```

Comment:
I was thinking the next 3 lines could stay forever, unless we figure out a way to make the tests run faster.